websub-ballerina/build.gradle

```
@@ -396,6 +383,4 @@ ballerinaPublish.dependsOn ballerinaBuild
 ballerinaPublish.dependsOn updateTomlFile
 ballerinaPublish.dependsOn initializeVariables
 ballerinaPublish.finalizedBy revertTomlFile
-publish.dependsOn ballerinaPublish
-
-createTestableJarZip.dependsOn ballerinaBuild
+publish.dependsOn ballerinaPublish
```

Shall we check the IDE settings for adding the empty line at the end of the files?

```suggestion
publish.dependsOn ballerinaPublish

```